sdk/core/azure-core/CHANGELOG.md

```diff
@@ -3,14 +3,19 @@
 ## 1.8.0-beta.4 (Unreleased)
 
 ### Features Added
+- Aligned OpenTelemetry attributes to align with OpenTelemetry 1.17.0.
 
 ### Breaking Changes
 
+- Removed the 3 parameter constructor for `Azure::Core::_internal::TracingContextFactory` and replaced it with a 4 parameter version which takes
```

Comment:
We should not be listing breaking internal changes, unless they are visible to customer as "broken compatibility with Attestation 1.0.0".

Reply:
What is the mechanism for notifying service authors of breaking changes to Azure Core.

Isn't that the point of the CHANGELOG - to call out changes that may affect customers?

Reply:
Our customers don't and should not consume things that are in `_internal` namespace.
Partner teams - yes, but we don't put these changes in the changelogs.

Reply:
What is the durable mechanism used to inform partner teams about changes made to core that are going to affect them? I would have thought that was the changelog, but apparently there's some other mechanism?